js/src/block-editor/index.js

```diff
@@ -3,15 +3,29 @@
 /**
  * WordPress dependencies
  */
-import { setLocaleData } from '@wordpress/i18n';
+import domReady from '@wordpress/dom-ready';
 import { addFilter } from '@wordpress/hooks';
+import { setLocaleData } from '@wordpress/i18n';
 
 /**
  * Internal dependencies
  */
 import { addControls, registerBlocks } from './helpers';
 import { Edit } from './components';
+import { GAClient } from '../common/classes';
 
 setLocaleData( { '': {} }, 'genesis-custom-blocks' );
 addFilter( 'genesisCustomBlocks.controls', 'genesisCustomBlocks/addControls', addControls );
+
+// @ts-ignore
 registerBlocks( genesisCustomBlocks, gcbBlocks, Edit );
+
+domReady( () => {
```

Comment:
`domReady` is interesting. Where you coming across any issues that needed this?

Reply:
That's a good point, I didn't notice that it was needed.

Moving it out of `domReady()` now…

Reply:
Moved out in https://github.com/studiopress/genesis-custom-blocks/pull/101/commits/87d1f456f42c97069065698e45448101c43215b7